spec/features/guest_can_visit_item_index_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe "As a visitor" do
+	xit "can visit the dashboard and see the link to item index" do
+		visit "/"
+		expect(page).to have_content("All Items")
+	end
+	it "from the root, visitor can visit items index and see all items" do
+		one_url = "http://pandathings.com/wp-content/uploads/2016/10/onesie-6-300x300.png"
+		two_url = "https://www.kcmaustralia.com/media/Animal%20Onesie/adventure%20time%20jake%20onesie.jpg"
+		item_one = Item.create(title: "Funsie Onesie", description: "number one", price: 800,
```

Comment:
I believe the price coming in should be 8.00 instead of 800. I checked in an IRB session:

```ruby
irb(main):001:0> 800.to_f
=> 800.0
irb(main):002:0> 8.to_f
=> 8.0
```